src/components/Capture/index.js

```diff
@@ -58,7 +58,7 @@ export default class Capture extends Component {
 
   handleMessages = (message) => {
     const { actions } = this.props
-    if (message.is_document) {
```

Comment:
so the backend has already changed to this signature?
And is the backend still backwards compatible?